scripts/create-user.sh

```diff
@@ -1,337 +0,0 @@
-#!/bin/bash
-set -euo pipefail
-SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
-SCRIPT=$0
-
-# shellcheck disable=SC1090
-source "${SCRIPT_DIR}/common.sh"
-
-###########################################################################
-# Defaults
-TEMPLATES_DIR="${SCRIPT_DIR}/templates"
-DEFAULT_SPACE=sandbox
-FROM_ADDRESS='gov-uk-paas-support@digital.cabinet-office.gov.uk'
-SUBJECT_CREATE='Create your GOV.UK PaaS account'
-SUBJECT_RESET='Reset your password on GOV.UK PaaS'
-NOTIFICATION='
-As a new account has been created please remember to update the
-gov-uk-paas-announce mailing list. You can do that by inviting the user to the
-group by using this URL:
-
-https://groups.google.com/a/digital.cabinet-office.gov.uk/forum/#!managemembers/gov-uk-paas-announce/invite
-
-As a welcome message you can use the text from here:
-https://groups.google.com/a/digital.cabinet-office.gov.uk/forum/#!forum/gov-uk-paas-announce
```

Comment:
With the update to the process, I don't see where we invite the user to the announce list.  Having said that, This feels like a gap in the process for users that are invited by an org manager anyway. This needs some thought.

Reply:
Indeed because actually I think I have forgotten this part of the process for a lot of tenants. 

We should automate that process.

Reply:
Yeah - I just missed this bit and would have forgotten it. I'll raise another PR to add these instructions back into the script (after the other prompt)